src/site/content/en/progressive-web-apps/manifest-updates/index.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: How Chrome handles updates to the web app manifest
```

Comment:
Your call. I wonder why I didn't ask for this to be shortened to "How Chrome updates the web app manifest". 

Generally, I think shorter is better for our titles. It's in our best interested to not make titles that cut off in small windows. This avoids inflating our bounce rate from readers clicking in to read the whole title.

Reply:
+1

In the long run we'd rename to "How browsers update the web app manifest"